### ISSUE TITLE
文中的准确率问题

### ISSUE THREAD

**Issue by JiamingZhuo** [qncsn2016/DeepGWC#1]
感谢您文中的新思路！我在运行此代码时出现了问题。对比运行结果和表2中DeepGWC的结果，表2中的结果是否为该代码中的BestAcc，而不是平均值？

**Comment by qncsn2016** [comment]
Yes, the results in Table 2 are corresponding to the best results in Table 3. The hyper-parameters of Table 2 are listed below it.

**Comment by JiamingZhuo** [comment]
感谢回答。从代码中我发现，DeepGWC取迭代最优准确率作为结果，实际模型收敛准确率并不及该值。是否可以理解为：文中用训练过程中最佳测试准确率，对比GCNII文中的收敛准确率平均值？

**Comment by qncsn2016** [comment]
Because of the different parameters, the model would oscillate in the later period of training. So we recorded the best accuracy rather than the range of accuracy like GCNII. In addition, DeepGWC is an incremental work on the basis of GCNII.

**Comment by ASan1527** [comment]
我想请教一下作者，您好，我想问下
cora_support01.npy中边的权值怎么设定的，还有cora_support10.npy表示的是哪些数据呢

**Comment by qncsn2016** [comment]
小波基是参考《Graph wavelet neural network》（GWNN）这篇论文计算的，可以参考GWNN的代码（https://github.com/Eilene/GWNN）
